packages/sdk-v2/src/sdk/analytics/google/types.ts

```diff
@@ -11,6 +11,6 @@ export interface IGoogleAPI {
   logEvent(event: string, data?: any): void;
 }
 
-export const defaultConfig: Partial<IGoogleConfig> = {
-  userProperty: 'user'
-}
+export const defaultGoogleConfig: Partial<IGoogleConfig> = {
```

Comment:
revert to `defaultConfig`. just do not re-export it in 'index.ts' barrel. it's for internal usage only

Reply:
done